app-kotlin/src/main/res/layout/activity_main.xml

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
+<com.bluelinelabs.conductor.ChangeHandlerFrameLayout
+    android:id="@+id/home_container"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/main_layout"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    >
-
-</FrameLayout>
+    tools:context=".MainActivity"/>
```

Comment:
why do we need to bind this layout to the activity? 🤔

Reply:
thanks for pointing this out I had resolved this.